api/tests/routes/pro/patch_collective_offer_test.py

```diff
@@ -359,6 +391,109 @@ def test_patch_offer_with_none_description(self, client):
         # Then
         assert response.status_code == 400
 
+    def test_patch_collective_offer(self, client):
```

Comment:
(détail / nommage) Puisqu'ici tu testes le cas particulier d'une tentative de modification d'une offre dont une réservation a été validée, le nom du test devrait l'indiquer clairement : 

```suggestion
    def test_cannot_update_offer_with_used_booking(self, client):
```